datalad/cmd.py

```diff
@@ -238,7 +238,7 @@ def decide_to_log(v):
 
     def run(self, cmd, log_stdout=True, log_stderr=True, log_online=False,
             expect_stderr=False, expect_fail=False,
-            cwd=None, env=None, shell=None):
+            cwd=None, env=None, shell=None, inputstream=None):
```

Comment:
Let's do "stdin" to be consistent with the other args that use ..._stderr and so on.